src/components/Charts/ModelChart.js

```diff
@@ -26,116 +30,140 @@ const ModelChart = ({
+        'beds',
+        duration,
+        'black',
+        'Available hospital beds',
+      ),
+    ];
 
-  const noAction = {
-    name: INTERVENTIONS.LIMITED_ACTION,
-    type: 'areaspline',
-    data: data[0].data,
-    marker: {
-      symbol: 'circle',
-    },
-  };
-  const socialDistancing = {
-    name:
-      currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE
-        ? formatIntervention(
-            INTERVENTIONS.SHELTER_IN_PLACE,
-            ' (poor compliance)',
-          )
-        : formatIntervention(INTERVENTIONS.SOCIAL_DISTANCING),
-    type: 'areaspline',
-    data: data[2].data,
-    marker: {
-      symbol: 'circle',
-    },
-  };
-  const shelterInPlace = {
-    name:
-      currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE
-        ? formatIntervention(
-            INTERVENTIONS.SHELTER_IN_PLACE,
-            ' (strict compliance)',
-          )
-        : formatIntervention(INTERVENTIONS.SHELTER_IN_PLACE),
-    type: 'areaspline',
-    data: data[1].data,
-    marker: {
-      symbol: 'circle',
-    },
-  };
-  const wuhanStyle = {
-    name: formatIntervention(INTERVENTIONS.LOCKDOWN),
-    type: 'areaspline',
-    data: data[3].data,
-    marker: {
-      symbol: 'circle',
-    },
-  };
-  const availableBeds = {
-    name: 'Available hospital beds',
-    type: 'spline',
-    data: data[4].data,
-    marker: {
-      symbol: 'circle',
-    },
-  };
+    data = scenarioComparisonOverTime(200);
+
+    noAction = {
+      name: INTERVENTIONS.LIMITED_ACTION,
+      type: 'areaspline',
+      data: data[0].data,
+      marker: {
+        symbol: 'circle',
+      },
+    };
+    socialDistancing = {
+      name:
+        currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE
+          ? formatIntervention(
+              INTERVENTIONS.SHELTER_IN_PLACE,
+              ' (poor compliance)',
+            )
+          : formatIntervention(INTERVENTIONS.SOCIAL_DISTANCING),
+      type: 'areaspline',
+      data: data[2].data,
+      marker: {
+        symbol: 'circle',
+      },
+    };
+    shelterInPlace = {
+      name:
+        currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE
+          ? formatIntervention(
+              INTERVENTIONS.SHELTER_IN_PLACE,
+              ' (strict compliance)',
+            )
+          : formatIntervention(INTERVENTIONS.SHELTER_IN_PLACE),
+      type: 'areaspline',
+      data: data[1].data,
+      marker: {
+        symbol: 'circle',
+      },
+    };
+    wuhanStyle = {
+      name: formatIntervention(INTERVENTIONS.LOCKDOWN),
+      type: 'areaspline',
+      data: data[3].data,
+      marker: {
+        symbol: 'circle',
+      },
+    };
+    availableBeds = {
+      name: 'Available hospital beds',
+      type: 'spline',
+      data: data[4].data,
+      marker: {
+        symbol: 'circle',
+      },
+    };
+  }
 
   const options = useMemo(() => {
+    if (!interventionsModelIsValid) {
+      return {
+        modelHasBadData: true,
```

Comment:
This just adds a little explanation if the model has bad data and the chart won't be displayed.